eval/generate_evaluation_case.py

```diff
@@ -41,8 +41,11 @@ def main(
         framework (str): The agent framework to use
         model (str): The model ID to use
     """
-    repo_root = Path.cwd()
-    workflows_dir = repo_root / generated_workflow_dir
+    if not generated_workflow_dir:
+        repo_root = Path.cwd()
+        workflows_dir = repo_root / "generated_workflows/latest"
```

Comment:
We are not using `latest` anymore. Maybe it would a better idea to throw an exception if the user did not set `generated_workflow_dir`.